recipes/thrift-cpp/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: thrift-cpp
+  version: "0.9.3"
+
+source:
+  fn: thrift-0.9.3.tar.gz
+  url: http://archive.apache.org/dist/thrift/0.9.3/thrift-0.9.3.tar.gz
+  md5: 88d667a8ae870d5adeca8cb7d6795442
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - boost
```

Comment:
It looks like you need C++11.  Right now, we add that with the gcc conda package.  If you compile with this, you'll also need to add libgcc as a runtime requirement.  Though the name could be better, it is a collection of GCC-related runtimes: libstdc++, libgfortran, libgcc, libgomp, and maybe others I'm forgetting.  The plan is to split them up, but that's how it is now.